src/blueapi/service/interface.py

```diff
@@ -70,8 +70,6 @@ def messaging_template() -> MessagingTemplate | None:
         )
         template.connect()
         return template
-    else:
-        return None
```

Comment:
Is there a reason for this change? It was put there in the first place because we decided running without a message bus was okay, especially for development purposes.